Dockerfile.mips64le

```diff
@@ -0,0 +1,13 @@
+FROM loongnix/alpine:3.11
+
+LABEL maintainer="Tom Denham <tom@tigera.io>"
```

Comment:
I'm not sure if Tom wants to be the maintainer of this image

Reply:
Oh so sorry i have refer to the old version and forgot changing this line. Now i have follow the latest version.

Reply:
the image is from [#76](https://github.com/alpinelinux/docker-alpine/issues/76) for mips64le